phpstan-baseline.neon

```diff
@@ -0,0 +1,57 @@
+parameters:
+	ignoreErrors:
+		-
+			message: "#^Access to protected property Ibexa\\\\AdminUi\\\\Form\\\\Data\\\\ContentTranslationData\\:\\:\\$content\\.$#"
+			count: 4
+			path: src/bundle/Form/Extension/ContentEditType.php
+
+		-
+			message: "#^Cannot access property \\$identifier on Ibexa\\\\Contracts\\\\Core\\\\Repository\\\\Values\\\\ContentType\\\\FieldDefinition\\|null\\.$#"
+			count: 2
+			path: src/bundle/Form/Extension/ContentEditType.php
```

Comment:
This indicated that we are missing dependencies in composer.json

Reply:
Fixed